src/test/resources/features/mpam2/QA.feature

```diff
@@ -101,13 +101,11 @@ Feature: QA
   Scenario Outline: User tests the auto-assign functionality of different actions at QA using multiple user accounts
     And I create a MPAM case with "Official" as the Reference Type and move it to the "QA" stage
     And I load and claim the current case
-    And I record the user who completed the previous stages
-    And I logout of the application
-    And I am logged into "CS" as user "CAMERON"
+    And I switch to user "CAMERON"
```

Comment:
Do we maybe want to remove both of out Ten10 accounts from the pack? This could be replaced by DECS_USER since that tends to be in every team

Reply:
fixed